doc/release/release_notes.rst

```diff
@@ -17,6 +17,88 @@ Release Notes
         * :mod:`wiz.system`
         * :mod:`wiz.utility`
 
+    .. change:: changed
+
+        Updated :mod:`wiz.validator` to use custom definition validation instead
+        of a JSON Schema validation as it is significantly hindering the
+        performance when creating an instance of
+        :class:`wiz.definition.Definition`.
+
+        Added :func:`wiz.validator.validate_definition` to perform equivalent
+        tests in optimized time.
+
+    .. change:: changed
+
+        Updated code to use `ujson <https://pypi.org/project/ujson/>`_ instead
+        of the built-in :mod:`json` module to optimize the loading of
+        :term:`JSON` files.
+
+    .. change:: changed
+
+        Updated :class:`wiz.definition.Definition` construction to simplify
+        logic and optimize performance. It does not inherit from
+        :class:`collections.Mapping` anymore and does not require from registry
+        and definition location to be included in the mapping.
```

Comment:
So the data stays, but you split it into definition data (in your example 'identifier') and moved the location and registry out and renamed it? so that you can access data without having to sanitize it?

Reply:
So the data stays, but you split it into definition data (in your example 'identifier') and moved the location and registry out and renamed it? so that you can access data without having to sanitize it?

Reply:
Yep, that's correct

Reply:
Updated

Reply:
fixed